tests/Squidex.Domain.Apps.Core.Tests/Model/Contents/WorkflowJsonTests.cs

```diff
@@ -19,8 +24,33 @@ public void Should_serialize_and_deserialize()
             var workflow = Workflow.Default;
 
             var serialized = workflow.SerializeAndDeserialize();
+            var test = workflow.GetTransitions(Status.Archived);
 
             serialized.Should().BeEquivalentTo(workflow);
         }
+
+        [Fact]
+        public void Should_verify_roles_mapping_in_workflow_transition()
+        {
+            var source = new
+            {
+               Expression = "expression_1",
+               Role = "role_1"
+            };
+
+            var serializerSettings = new JsonSerializerSettings
+            {
+                ContractResolver = new ConverterContractResolver()
+            };
+
+            var json = JsonConvert.SerializeObject(source, serializerSettings);
+
+            var serialized = JsonConvert.DeserializeObject<JsonWorkflowTransition>(json);
```

Comment:
Plese use the SerializeAndDeserialize extension method for or the Helper class in general and not JSON.NET directly

Reply:
I've updated it, it's good to utilize extension methods throughout. Thanks.